src/sqlfluff/core/rules/analysis/select_crawler.py

```diff
@@ -78,7 +78,9 @@ def select_info(self):
 
     def get_wildcard_info(self) -> List[WildcardInfo]:
         """Find wildcard (*) targets in the SELECT."""
-        buff = []
+        buff: List[WildcardInfo] = []
+        if not self.select_info:
+            return buff
```

Comment:
Do you know what type of query or statement caused this? Worth adding a comment to explain when this may happen?

Reply:
Ditto